packages/fxa-admin-panel/src/index.test.tsx

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactNode } from 'react';
+window.console.log = jest.fn();
```

Comment:
Is this intentional? Does it need to be restored at some point?

Reply:
Hmm, I needed this at some point when I was working on this file, but the tests pass without it so I'll remove it, thanks for asking.